packages/microcosm-www-next/src/pages/index.js

```diff
@@ -14,77 +14,55 @@ export default class IndexPage extends React.Component {
     super(props)
     this.state = {
       currentSection: 0,
-      graphicsMap: [],
       microcosmView: true,
       numSections: Object.keys(data)
     }
   }
 
   componentDidMount() {
     this.setVars()
-    this.beginObserve()
+    this.bindEvents()
```

Comment:
if this gets called from other spots the cool, if not then maybe simplify by binding events here and then breaking it out into its own thing as needed.